python/lsst/meas/extensions/photometryKron/photometryKron.cc

```diff
@@ -0,0 +1,140 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ * See the COPYRIGHT file
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+#include <cstdint>
+
+#include <pybind11/pybind11.h>
+
+#include "lsst/pex/config/pybind11.h" // defines LSST_DECLARE_CONTROL_FIELD
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/Ellipses.h"
+#include "lsst/afw/geom/AffineTransform.h"
+#include "lsst/afw/image/MaskedImage.h"
+#include "lsst/afw/table/Source.h"
+#include "lsst/meas/base/Algorithm.h"
+#include "lsst/meas/extensions/photometryKron.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace extensions {
+namespace photometryKron {
+
+namespace {
+
+void declareKronFluxControl(py::module &mod) {
+    py::class_<KronFluxControl> cls(mod, "KronFluxControl");
+
+    cls.def(py::init<>());
+
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, fixed);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, nSigmaForRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, nIterForRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, nRadiusForFlux);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, maxSincRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, minimumRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, enforceMinimumRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, useFootprintRadius);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, smoothingSigma);
+    LSST_DECLARE_CONTROL_FIELD(cls, KronFluxControl, refRadiusName);
+}
+
+void declareKronFluxAlgorithm(py::module &mod) {
+    py::class_<KronFluxAlgorithm, base::SimpleAlgorithm> cls(mod, "KronFluxAlgorithm");
+
+    cls.attr("FAILURE") = py::cast(static_cast<int>(KronFluxAlgorithm::FAILURE));
```

Comment:
Might be a good idea. On the other hand we may be done wrapping such things, and I hope pybind11 will provide a cleaner solution at some point.